### PR TITLE
feat: Add PHP template

### PR DESCRIPTION
**Pull request by Sironheart**

### CODE REVIEW

File: template/php/flake.nix
Comment (hbjydev) [inline template/php/flake.nix:23]:
```suggestion
        php = php83.buildEnv {
```

Let's rename this to seem less placeholder-y.

**Review by hbjydev — CHANGES_REQUESTED** [review]
Nice PR! Thanks for the contrib. I'll approve the CI job to check this builds, just one nitpicky request for changes :)

File: template/php/flake.nix
Comment (Sironheart) [reply template/php/flake.nix]: Renamed it to `phpEnvironment`, since it seems fair enough to use this name :D 

Also renamed the php file in the src folder to match the webserver standard of file handling (`/index.php` == `/`)

**Review by hbjydev — APPROVED** [review]
LGTM! Will get this merged once I verify that it actually builds, CI is having... issues currently lol

**Review by hbjydev — CHANGES_REQUESTED** [review]
Okay, so. When trying to build this, it seems this doesn't output anything resembling a binary or script in the `/bin` path in the derivation output.

```bash
/tmp/tmp.mUjAv72oFJ via 🐘
❯ nix run .
error: unable to execute '/nix/store/pn5ag15jq0kkrw6iwd36hh74wwak9q5j-example-0.1.0/bin/example': No such file or directory
```

If this _won't_ create any form of entrypoint executable, that's fine, but we'll drop the `package` entries for it and make it a dev shell-only one. Unless you can think of a better implementation here?

Happy for you to implement this as a binary, but I'll not merge this in its current state as it 'builds' but has nothing to run.

We can chat more about it in Discord if I've been unclear at all here.

**Review by hbjydev — COMMENTED** [review]
Looks good! I'll re-review later on :)

**Pull request merged**
